src/pages/WriteProductsPage.tsx

```diff
@@ -9,59 +10,72 @@ type FormData = {
 
 export const WriteProductsPage: React.FC = () => {
 	const { register, handleSubmit, formState: { errors } } = useForm<FormData>();
+	const [text, setText] = useState('投稿する？');
 
 	const onSubmit: SubmitHandler<FormData> = (data) => {
 		console.log('Submitted Data:', data);
 		alert('投稿できました！！！');
 	};
 
 	return (
-		<div>
-			<h1>Write Products Page</h1>
-			<form onSubmit={handleSubmit(onSubmit)}>
-				{/* タイトル入力フィールド */}
-				<div>
-					<label htmlFor='title'>タイトル</label>
-					<input
-						id='title'
-						{...register('title',{
-							required: 'タイトルは必須です'
-						})}
-						placeholder='タイトルを入力してください'
-					/>
-					{errors.title && <p style={{ color: 'red' }}>{errors.title.message}</p>}
-				</div>
+		<div className='flex justify-center bg-gray-100'>
+			<div className='w-full max-w-xl'>
+				<h1 className='text-lg font-bold text-gray-800 mt-4 mb-4 text-center'>Write Products Page</h1>
+				<form className='bg-white shadow-md rounded px-8 pt-6 pb-8 mb-8' onSubmit={handleSubmit(onSubmit)}>
+					{/* タイトル入力フィールド */}
+					<div>
+						<label className='block text-gray-700 text-sm font-bold mb-2' htmlFor='title'>タイトル</label>
+						<input
+							className={`border shadow appearance-none rounded w-full py-2 px-3 text-gray-700 mb-2 leading-tight focus:outline-none focus:shadow-outline ${errors.title ? 'border-red-500' : 'border-grey-100'}`}
+							id='title'
+							{...register('title',{
+								required: 'タイトルは必須です'
+							})}
+							placeholder='タイトルを入力してください'
+						/>
+						{errors.title && <p className='absolute text-red-500 text-xs italic'>{errors.title.message}</p>}
+					</div>
```

Comment:
```suggestion
					<div className='text-gray-700'>
						<label className='block text-sm font-bold mb-2' htmlFor='title'>タイトル</label>
						<input
							className={`border shadow appearance-none rounded w-full py-2 px-3 mb-2 leading-tight focus:outline-none focus:shadow-outline ${errors.title ? 'border-red-500' : 'border-grey-100'}`}
							id='title'
							{...register('title',{
								required: 'タイトルは必須です'
							})}
							placeholder='タイトルを入力してください'
						/>
						{errors.title && <p className='absolute text-red-500 text-xs italic'>{errors.title.message}</p>}
					</div>
```
こんな感じに、テキストの色が同じところは上位のdivタグに組んで、エラーとかの一部色を変えたいところで改めて色を変えても、同じように動いたはず。